test/membrane_rtc_engine/media_event_test.exs

```diff
@@ -3,6 +3,51 @@ defmodule Membrane.RTC.Engine.MediaEventTest do
 
   alias Membrane.RTC.Engine.MediaEvent
 
+  describe "create_peer_joined_event" do
+    test "" do
```

Comment:
Maybe we could remove `describe` block as we have only one test inside it?
We could then name this test

```elixir
test "peer_joined event is created properly"
```

I usally read this as `test if peer_joined event is created properly" but I am not sure what is the convention so we can leave it as it is too

Reply:
Oh, looks like I forgot to add a real string to the test itself. 

I can take out the describe blocks if you like. I'm just used to adding them to organize tests by function, but it's probably better to be consistent with the rest of the codebases.

Reply:
Ok :+1: So let's just add string to the `test`